### DISCUSSION TITLE
Unable to change or remove an asset after saving

### DISCUSSION THREAD

**Discussion by japgroevemaker** [statamic/cms#9922] [Troubleshooting]
Hello,

I'm having trouble with changing or removing an asset after saving the entry.

When i have uploaded the asset and didn't save the entry, everything works just fine
![image](https://github.com/statamic/cms/assets/22168803/e2398792-91c3-42e7-988c-7c8428ba020f)

But when i've saved the entry, the remove option is suddenly gone
![image](https://github.com/statamic/cms/assets/22168803/eb453042-5309-4cd6-88f9-8e42e510a8e1)

Can anyone help me with this? The statamic version is 3.3.68

**Comment by joshuablum** [comment]
Hey @japgroevemaker,

Does the user you're logged in as have the necessary permission to delete assets? https://statamic.dev/users#native-permissions

Also:

> The statamic version is 3.3.68

This version of Statamic is outdated and not supported anymore. I would recommend to upgrade to the latest version. Could potentially be a bug that got fixed since then.
Please see our upgrade guides for this: https://statamic.dev/upgrade-guide
